lib/packages/core/lib/src/utils/preferences/preferences.dart

```diff
@@ -23,63 +20,35 @@ class LMChatPreferences extends ILMPreferenceService {
   static LMChatPreferences? _instance;
   static LMChatPreferences get instance => _instance ??= LMChatPreferences._();
 
-  late SharedPreferences _sharedPreferences;
-
   LMChatPreferences._();
 
-  User? currentUser;
-
-  set setCurrentUser(User user) => currentUser = user;
-  get getCurrentUser => currentUser;
-
-  @override
-  Future<void> initialize() async {
-    _sharedPreferences = await SharedPreferences.getInstance();
-  }
+  get getCurrentUser => getUser();
 
   @override
   Future<void> storeUserData(User user) async {
-    setCurrentUser = user;
-    UserEntity userEntity = user.toEntity();
-    Map<String, dynamic> userData = userEntity.toJson();
-    String userString = jsonEncode(userData);
-    await _sharedPreferences.setString('user', userString);
+    await LMChatCore.instance.lmChatClient.deleteLoggedInUser();
+    await LMChatCore.instance.lmChatClient.insertOrUpdateLoggedInUser(user);
```

Comment:
please don't delete the user, it is irrelevant in real world scenario

Reply:
Removed.